scene/2d/remote_transform_2d.cpp

```diff
@@ -42,6 +42,32 @@ void RemoteTransform2D::_update_cache() {
 	}
 }
 
+Transform2D RemoteTransform2D::_create_updated_transform(Transform2D our_trans, Transform2D n_trans) const {
+	if (update_remote_rotation) {
+		if (!update_remote_position) {
+			our_trans.set_origin(n_trans.get_origin());
+		}
+		if (!update_remote_scale) {
+			our_trans.set_scale(n_trans.get_scale());
+		}
+		if (!update_remote_skew) {
+			our_trans.set_skew(n_trans.get_skew());
+		}
```

Comment:
I tested the elapsed time of these methods: `set_scale()` << `set_rotation_scale_and_skew()` < `set_skew()`

That is, if you don't just call `set_scale()`, it is better to call `set_rotation_scale_and_skew()` directly. You can single out the case where **only** `set_scale()` is called.